src/components/sbb-radio-button/sbb-radio-button.tsx

```diff
@@ -159,15 +158,14 @@ export class SbbRadioButton implements ComponentInterface {
         aria-checked={`${this.checked}`}
         aria-disabled={`${this.disabled || this._disabledFromGroup}`}
         /* eslint-enable jsx-a11y/aria-proptypes */
-        aria-labelledby={this._radioButtonLabelId}
         role="radio"
       >
-        <label id={this._radioButtonLabelId} htmlFor={this.radioButtonId} class="sbb-radio-button">
+        <label id={this._radioButtonLabelId} htmlFor={this._radioButtonId} class="sbb-radio-button">
```

Comment:
As the label is wrapped around the input, you don't need _radioButtonId anymore (sorry wrong advice of me before.)